lib/cassia/requests/close_ap_state.rb

```diff
@@ -0,0 +1,31 @@
+module Cassia
+  module Requests
+    class CloseAPState
```

Comment:
I think this should be `CloseApState` just to aline with the naming of the file `close_ap_state`